lib/extensions.js

```diff
@@ -509,34 +509,51 @@ module.exports = function (realmConstructor) {
       Connected: "connected",
     };
 
+    realmConstructor.ConnectionState = realmConstructor.App.Sync.ConnectionState;
+
     realmConstructor.App.Sync.SessionState = {
       Invalid: "invalid",
       Active: "active",
       Inactive: "inactive",
     };
 
+    realmConstructor.SessionState = realmConstructor.App.Sync.SessionState;
+
     realmConstructor.App.Sync.ProgressDirection = {
       Download: "download",
       Upload: "upload",
     };
 
+    realmConstructor.ProgressDirection = realmConstructor.App.Sync.ProgressDirection;
+
     realmConstructor.App.Sync.ProgressMode = {
       ReportIndefinitely: "reportIndefinitely",
       ForCurrentlyOutstandingWork: "forCurrentlyOutstandingWork",
     };
 
+    realmConstructor.ProgressMode = realmConstructor.App.Sync.ProgressMode;
+
     realmConstructor.App.Sync.ClientResyncMode = {
```

Comment:
I think this should be `ClientResetMode`.  `ClientResyncMode` doesn't exist in the `.d.ts`

Reply:
Thanks :+1: